project/build.properties

```diff
@@ -1 +0,0 @@
-sbt.version = 0.13.12
```

Comment:
I don't think it is necessary to explicitly specify a sbt-version in this project.